java/flight/flight-core/src/main/java/org/apache/arrow/flight/ServerSessionMiddleware.java

```diff
@@ -16,8 +16,10 @@
  */
 package org.apache.arrow.flight;
 
+import com.google.common.base.Splitter;
```

Comment:
Ideally we wouldn't use Guava? I suppose as long as this interface seems stable...

Is there an apache-commons equivalent we can use?

Reply:
I will take a look, this was actually suggested by the ErrorProne library, it wasn't my first choice.